extensions/ql-vscode/gulpfile.ts/package.ts

```diff
@@ -9,7 +9,7 @@ export async function packageExtension(): Promise<void> {
     'package',
     '--out', path.resolve(deployedPackage.distPath, '..', `${deployedPackage.name}-${deployedPackage.version}.vsix`)
   ];
-  const proc = child_process.spawn('vsce', args, {
+  const proc = childProcess.spawn('npx', ['vsce'].concat(args), {
```

Comment:
I'd like to avoid `npx` and use the package-installed version of `vsce`.

Reply:
Agree as above.